codecept.json

```diff
@@ -0,0 +1,34 @@
+{
+  "output": "./output",
+  "helpers": {
+    "WebDriverIO": {
+      "url": "https://dev.planx-pla.net",
+      "smartWait": 5000,
+      "browser": "chrome",
+      "restart": false,
+      "windowSize": "normal",
+      "timeouts": {
+        "script": 6000,
+        "page load": 10000
+      }
+    },
+    "REST": {
+      "endpoint": "https://dev.planx-pla.net",
```

Comment:
This should be configurable - we want to run this test against different QA environments